content/blog/2019/06/2019-06-14-performance-testing-jenkins.adoc

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Performance Testing Framework for Jenkins Plugins"
+tags:
+- jenkins
+- jmh
+- plugins
+- benchmark
+- performance
+- gsoc
+author: abhyudayasharma
+---
+
+= Performance Testing Framework for Jenkins Plugins
+
+My work for the first phase of my Google Summer of Code project to improve the performance of
+the Role Strategy Plugin involved creating a framework for running benchmarks in Jenkins Plugins.
+We chose link:http://openjdk.java.net/jeps/230[Java Microbenchmark Harness] for running the benchamrks.
```

Comment:
Would be nice to expand it a bit to highlight the community value. Then the technology side could go to the second paragraph along with the reference to releases which include the framework